helpers/constants.go

```diff
@@ -223,6 +223,12 @@ const (
 	PIPlacementGroupPolicy = "pi_placement_group_policy"
 	PIPlacementGroupID     = "pi_placement_group_id"
 
+	// IBM PI Workspace
```

Comment:
Let's avoid adding Terraform related const here.

Reply:
I cannot.

Reply:
No, we have stopped that practice. terraform variables are used there Itself. These are older vars that can also be moved there.
Doesn't make sense to relate this repo with Terraform project only.

Reply:
@yussufsh  it has been moved. Could you review ?